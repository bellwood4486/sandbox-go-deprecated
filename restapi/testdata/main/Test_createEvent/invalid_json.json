This is invalid json format.
{
  "ID": "2",
  "Title": "Test Title",
  "Description": "Test Description"
}