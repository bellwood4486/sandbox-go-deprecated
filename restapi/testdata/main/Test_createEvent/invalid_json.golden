{
  "Kind": "invalid_json",
  "Message": "invalid character 'T' looking for beginning of value"
}